integration-tests/src/test/java/fr/pilato/elasticsearch/crawler/fs/test/integration/FsCrawlerRestIT.java

```diff
@@ -74,6 +74,42 @@ public void testAllDocumentsWithRest() throws Exception {
         }
     }
 
+    @Test
+    public void testDocumentWithCustomTagsAndWithRest() throws Exception {
+        Path from = rootTmpDir.resolve("resources").resolve("documents");
+        if (Files.notExists(from)) {
+            staticLogger.error("directory [{}] should exist before wa start tests", from);
+            throw new RuntimeException(from + " doesn't seem to exist. Check your JUnit tests.");
+        }
+
+        Path tagsFilePath = from.getParent().resolve("fscrawler-test-upload-with-tags.txt").toAbsolutePath();
```

Comment:
Could you move your test file to a dir named `tags` for example?
And here write:

```java
Path tagsFilePath = rootTmpDir.resolve("resources").resolve("tags").resolve("fscrawler-test-upload-with-tags.txt").toAbsolutePath();
```